opentelemetry-sdk/tests/.benchmarks/Linux-CPython-3.5-64bit/0001_old.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Should probably add a [`linguist-generated` entry](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/customizing-how-changed-files-appear-on-github) for these so to keep PR noise down?

Reply:
That's really cool! I didn't know about this 😄 

However I think it's only big this time, the benefit of having it included in the diff is that when someone updates the performance later, only the numbers should change (which means we can see how the performance changes in the git diff)

Also if you add new performance tests, that will be clearly seen as only a few lines of code added to these files.

Let me know what you think?

Reply:
Oh I see, it always updates the same file. I was looking at the pytest-benchmark docs [here](https://pytest-benchmark.readthedocs.io/en/latest/comparing.html) with `--benchmark-autosave`. Is it worth preserving all previous runs? They show some cool box and whisker plots from previously saved runs. We can always iterate 😄

Reply:
It's a good idea! For now we will have a history of the mean throughput in `benchmarks/data.js` for all commits, but if someone wants to generate the cool whisker plots all they would have to do is clone the repo and run `tox -e tests-core-sdk -- --benchmark-histogram=<path>` to generate the `.json`.

But I think it would be cool to have just the latest `.json` file committed somewhere here on the repo later :)